docs/resources/gaussdb_redis_instance.md

```diff
@@ -0,0 +1,180 @@
+  For details, see [DB Instance Specifications](https://support.huaweicloud.com/intl/en-us/redisug-nosql/nosql_05_0059.html)
+
+* `password` - (Required, String) Specifies the database password. The value must be 8 to 32 characters
+  in length, including uppercase and lowercase letters, digits, and special characters,
+  such as ~!@#%^*-_=+? You are advised to enter a strong password to improve security, preventing security risks
+  such as brute force cracking.
+
+* `vpc_id` -  (Required, String, ForceNew) Specifies the VPC ID.
+  Changing this parameter will create a new resource.
+
+* `subnet_id` - (Required, String, ForceNew) Specifies the network ID of a subnet.
+  Changing this parameter will create a new resource.
+
+* `security_group_id` - (Optional, String) Specifies the security group ID. 
+  Required if the selected subnet doesn't enable network ACL.
+  Do nothing in update method if change this parameter.
+
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the enterprise project id, Only valid for users who
+  have enabled the enterprise multi-project service.
+  Changing this parameter will create a new resource.
+
+* `force_import` - (Optional, Bool) If specified, try to import the instance instead of creating if the name already existed.
+
+* `datastore` - (Optional, List, ForceNew) Specifies the database information. Structure is documented below.
+  Changing this parameter will create a new resource.
+
+* `backup_strategy` - (Optional, List) Specifies the advanced backup policy. Structure is documented below.
+  Do nothing in update method if change this parameter.
+
+* `tags` - (Optional, Map) The key/value pairs to associate with the instance.
+
+The `datastore` block supports:
+
+* `engine` - (Optional, String, ForceNew) Specifies the database engine. Only "redis" is supported now.
+
+* `version` - (Optional, String, ForceNew) Specifies the database version. Only "5.0" is supported now.
+
+* `storage_engine` - (Optional, String, ForceNew) Specifies the storage engine. Only "rocksDB" is supported now.
+
+
+The `backup_strategy` block supports:
+
+* `start_time` - (Required, String) Specifies the backup time window. Automated backups
+  will be triggered during the backup time window. It must be a valid value in
+  the "hh:mm-HH:MM" format. The current time is in the UTC format.
+  The HH value must be 1 greater than the hh value. The values of mm and MM
+  must be the same and must be set to 00. Example value: 08:00-09:00, 03:00-04:00.
+
+* `keep_days` - (Optional, Int) Specifies the number of days to retain the generated
+   backup files. The value ranges from 0 to 35.
+   If this parameter is set to 0, the automated backup policy is not set.
+   If this parameter is not transferred, the automated backup policy is enabled by default.
+   Backup files are stored for seven days by default.
+
+* `charging_mode` - (Optional, String) Specifies the charging mode of the GaussDB for Redis instance.
```

Comment:
please put the following arguments(charging_mode, period_unit, period and auto_renew) in front of blocks.

Reply:
fixed